test/integration/load_stats_integration_test.cc

```diff
@@ -566,6 +571,7 @@ TEST_P(LoadStatsIntegrationTest, NoLocalLocality) {
 
 // Validate the load reports for successful/error requests make sense.
 TEST_P(LoadStatsIntegrationTest, Error) {
+  XDS_DEPRECATED_FEATURE_TEST_SKIP;
```

Comment:
Aren't we losing a lot of coverage everywhere we are skipping tests? Maybe I'm confused about what is being done here?

Reply:
This only applies when `--define deprecated_features=disabled` is set, so not in regular builds.

Reply:
Ah OK, I see. Thanks.